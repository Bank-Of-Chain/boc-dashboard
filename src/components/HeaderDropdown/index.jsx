import React from 'react'
import PropTypes from 'prop-types'
import { Dropdown } from 'antd'
import classNames from 'classnames'

const HeaderDropdown = ({ overlayClassName: cls, ...restProps }) => <Dropdown overlayClassName={classNames(cls)} {...restProps} />

HeaderDropdown.propTypes = {
  overlayClassName: PropTypes.string
}

export default HeaderDropdown
