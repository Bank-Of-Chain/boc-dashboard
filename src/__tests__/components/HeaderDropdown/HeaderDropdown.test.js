import React from 'react'
import { render } from '@testing-library/react'
import HeaderDropdown from '@/components/HeaderDropdown/index'

test('HeaderDropdown Component Render', () => {
  const { asFragment } = render(
    <HeaderDropdown>
      <span>123</span>
    </HeaderDropdown>
  )
  expect(asFragment()).toMatchSnapshot()
})
