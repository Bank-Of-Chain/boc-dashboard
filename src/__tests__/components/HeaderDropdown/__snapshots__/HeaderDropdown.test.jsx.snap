// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HeaderDropdown Component Render 1`] = `
<DocumentFragment>
  <button
    class="ant-btn css-dev-only-do-not-override-yp8pcc ant-btn-default ant-dropdown-trigger"
    type="button"
  >
    <span>
      hover menu
    </span>
  </button>
</DocumentFragment>
`;
